src/main/java/Parser.java

```diff
@@ -0,0 +1,39 @@
+import java.util.Scanner;
+
+public class Parser {
+
+    public Parser() {
+    }
+
+    public String checkCommand(String command) throws DukeInvalidCommandException, DukeIncompleteCommandException {
+        if (command.split(" ").length != 1) {
+            return getCommand(command);
+        }
+
+        String wrongCommand = getCommand(command);
```

Comment:
Perhaps you can add a validation check to check if the input is completely empty.